.gitignore

```diff
@@ -1,6 +1,6 @@
 .idea/
 .pytest_cache/
-
+config/configuration.json
```

Comment:
this belongs down near line 106, in the section provided for configuration files. Minor nitpick